crates/aptos-build-info/Cargo.toml

```diff
@@ -0,0 +1,12 @@
+[package]
```

Comment:
nit: I'd copy an existing Cargo.toml file. There's a lot of standard information missing from this, e.g.:
```
[package]
name = "aptos"
version = "0.3.0"
authors = ["Aptos Labs <opensource@aptoslabs.com>"]
description = "Aptos tool for management of nodes and interacting with the blockchain"
repository = "https://github.com/aptos-labs/aptos-core"
homepage = "https://aptoslabs.com"
license = "Apache-2.0"
publish = false
edition = "2018"
```

Reply:
Done.